modules/library/src/main/scala/zio/elasticsearch/HttpElasticExecutor.scala

```diff
@@ -154,6 +167,24 @@ private[elasticsearch] final class HttpElasticExecutor private (config: ElasticC
     }
   }
 
+  private def executeCreatePointInTime(index: IndexName): Task[(Chunk[Item], Option[(String, Option[Json])])] =
```

Comment:
We should consider what should we use, Chunk or List.

Reply:
ZStream method paginateChunkZIO expects a Chunk so we must use it here.